mongo/integration/crud_helpers_test.go

```diff
@@ -275,7 +275,6 @@ func executeInsertOne(mt *mtest.T, sess mongo.Session, args bson.Raw) (*mongo.In
 	for _, elem := range elems {
 		key := elem.Key()
 		val := elem.Value()
-
```

Comment:
We do have weird new lines throughout the driver (mostly from previously code-generated code), but this one seems fine to keep.